dev-requirements.txt

```diff
@@ -28,6 +28,7 @@ iniconfig==1.1.1
 isort==5.10.1
 itsdangerous==2.1.2
 Jinja2==3.1.1
+jsonschema==3.2.0
```

Comment:
This seems to be used by some that we use. So it gets auto-generated on doing: `pip freeze > dev-requirements.txt`

You can safely ignore this.